src/renderer/state/upload/logics.ts

```diff
@@ -755,7 +760,7 @@ const uploadWithoutMetadataLogic = createLogic({
                 user,
                 {
                   groupId,
-                  multifile: isMultifile
+                  multifile: false // because we disallow uploading multifiles without metadata at all
```

Comment:
I agree that we should wholesale disallow uploads w/out metadata.  Can we identify a product owner / stakeholder list to dedide? @TravisKroeker ?